manifests/a/Apple/iTunes/12.12.5.8/Apple.iTunes.installer.yaml

```diff
@@ -0,0 +1,38 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Apple.iTunes
+PackageVersion: 12.12.5.8
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+  Custom: /norestart
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://secure-appldnld.apple.com/itunes12/012-67782-20220909-091C2557-7D04-4926-AD36-CE1EC201A92D/iTunes64Setup.exe
+  InstallerSha256: 97584C3E1E3D7CEDEC144915DDF516ADFABF4E8219BCA60FD02961EBEEDBB46C
+  AppsAndFeaturesEntries:
+  - DisplayName: iTunes
+    Publisher: Apple Inc.
+    ProductCode: '{781FFA26-A8FF-47B7-9BFB-5F4FB3A72315}'

```

Comment:
```suggestion
    ProductCode: '{E42D1B4D-78C2-4144-8E4D-FA82C53B68A4}'
```

Reply:
How dod you get this value, so that i can make it easier for you next time